src/ExactDiagonalization/exact_diagonalization_problem.jl

```diff
@@ -12,6 +12,10 @@ collection of addresses can be passed as `v0`.
 - `algorithm=LinearAlgebraSolver()`: The algorithm to use for solving the problem. The
     algorithm can also be specified as the second positional argument in the `init`
     function.
+- `linear_dimension` (optional): The estimated dimension of the problem. This is
+    usually automatically determined from the Hamiltonian.
+- `info=false`: Whether to print additional information about memory requirements.
+- `warn=true`: Whether to print warnings if available memory is insufficient.
```

Comment:
```suggestion
- `memory_info=false`: Whether to print additional information about memory requirements.
- `memory_warn=true`: Whether to print warnings if available memory is insufficient.
```
Not sure about the names but if these are only used to report memory-related information, that should be reflected.